libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -294,27 +399,53 @@
         "comment": "Do not translate HTML markup enclosed in <>. Do not translate {icons}. Translate words 'segment by'.",
         "limit": 0
     },
+    "dashboard.bucket.metric_segment_by_warning|metric": {
+        "value": "To add additional metric, remove {icons} from <span class=\"stack-by\">segment by</span>",
+        "comment": "Do not translate HTML markup enclosed in <>. Do not translate {icons}. Translate words 'segment by'.",
+        "limit": 0
+    },
     "dashboard.bucket.metric_stack_by_warning": {
         "value": "To add additional measure, remove {icons} from '<span class=\"stack-by\">'stack by'</span>'",
         "comment": "Do not translate HTML markup enclosed in <>. Do not translate {icons}. Translate words 'stack by'.",
         "limit": 0
     },
+    "dashboard.bucket.metric_stack_by_warning|metric": {
+        "value": "To add additional metric, remove {icons} from <span class=\"stack-by\">stack by</span>",
+        "comment": "Do not translate HTML markup enclosed in <>.  Do not translate {icons}. Translate words 'stack by'.",
+        "limit": 0
+    },
     "dashboard.bucket.metric_view_by_warning": {
         "value": "To add additional measure, remove {icons} from '<span class=\"stack-by\">'view by'</span>'",
         "comment": "Do not translate HTML markup enclosed in <>. Do not translate {icons}. Translate words 'view by'.",
         "limit": 0
     },
+    "dashboard.bucket.metric_view_by_warning|metric": {
+        "value": "To add additional measure, remove {icons} from <span class=\"stack-by\">view by</span>",
```

Comment:
`measure` here should be changed to `metric`, shouldn't it?

Reply:
fixed